File: Elements Of Programming/Quadratic.java
public class Quadratic {
    public static void main(String[] args) {
        double b = Double.parseDouble(args[0]);
        double c = Double.parseDouble(args[1]);

        double disc = b * b - 4.0 * c;
        double d = Math.sqrt(disc);

        System.out.println((-b + d) / 2.0);
        System.out.println((-b - d) / 2.0);
    }
}

File: elements-of-programming/quadratic.py
import math
import sys


b = float(sys.argv[1])
c = float(sys.argv[2])

if c == 0:
    print('Error: division by zero!')
else:
    discriminant = b * b - 4.0 * c
    d = math.sqrt(discriminant)
    print((-b + d) / 2.0)
    print((-b - d) / 2.0)
